.travis.yml

```diff
@@ -8,3 +8,4 @@ script:
  - cd src
  - dotnet restore
  - dotnet build -c Release
+ - dotnet test Functional.CQS.AOP.IoC.SimpleInjector.Tests/Functional.CQS.AOP.IoC.SimpleInjector.Tests.csproj
```

Comment:
Is there a way to get this to run for every .Tests project automatically?

For instance, I've seen `Dockerfile`s that have the following line:
```
RUN for f in ./**/*.Tests.csproj; do dotnet test $f; done
```

Reply:
It took a bit of digging, but apparently `dotnet test` supports "run all tests in solution" use case now

https://github.com/Microsoft/vstest/issues/705

Reply:
I am much happier with the new build configuration that is future-proofed against the addition of new test projects.  Thanks for pointing this out @willikev !